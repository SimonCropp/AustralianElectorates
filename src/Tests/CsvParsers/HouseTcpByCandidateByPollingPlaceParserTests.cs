using System.Linq;
using ObjectApproval;
using Xunit;
using Xunit.Abstractions;

public class HouseTcpByCandidateByPollingPlaceParserTests :
    XunitLoggingBase
{
    [Fact]
    public void Run()
    {
        ObjectApprover.VerifyWithJson(HouseTcpByCandidateByPollingPlaceParser.Read("CsvParsers/HouseTcpByCandidateByPollingPlaceSample.csv").Take(10));
    }

    public HouseTcpByCandidateByPollingPlaceParserTests(ITestOutputHelper output) :
        base(output)
    {
    }
}