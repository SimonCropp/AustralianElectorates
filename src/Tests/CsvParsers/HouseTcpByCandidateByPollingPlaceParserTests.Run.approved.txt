[
  {
    PollingPlaceID: 93925,
    DivisionNm: 'Bean',
    Elected: "N",
    HistoricElected: "N",
    PartyAb: 'LP',
    OrdinaryVotes: 288,
    Swing: 0.93
  },
  {
    PollingPlaceID: 93925,
    DivisionNm: 'Bean',
    Elected: "Y",
    HistoricElected: "N",
    PartyAb: 'ALP',
    OrdinaryVotes: 418,
    Swing: -0.93
  },
  {
    PollingPlaceID: 93927,
    DivisionNm: 'Bean',
    Elected: "N",
    HistoricElected: "N",
    PartyAb: 'LP',
    OrdinaryVotes: 2,
    Swing: -34.62
  },
  {
    PollingPlaceID: 93927,
    DivisionNm: 'Bean',
    Elected: "Y",
    HistoricElected: "N",
    PartyAb: 'ALP',
    OrdinaryVotes: 11,
    Swing: 34.62
  },
  {
    PollingPlaceID: 11877,
    DivisionNm: 'Bean',
    Elected: "N",
    HistoricElected: "N",
    PartyAb: 'LP',
    OrdinaryVotes: 504,
    Swing: 0.62
  },
  {
    PollingPlaceID: 11877,
    DivisionNm: 'Bean',
    Elected: "Y",
    HistoricElected: "N",
    PartyAb: 'ALP',
    OrdinaryVotes: 826,
    Swing: -0.62
  },
  {
    PollingPlaceID: 11452,
    DivisionNm: 'Bean',
    Elected: "N",
    HistoricElected: "N",
    PartyAb: 'LP',
    OrdinaryVotes: 794,
    Swing: 1.99
  },
  {
    PollingPlaceID: 11452,
    DivisionNm: 'Bean',
    Elected: "Y",
    HistoricElected: "N",
    PartyAb: 'ALP',
    OrdinaryVotes: 1092,
    Swing: -1.99
  },
  {
    PollingPlaceID: 8761,
    DivisionNm: 'Bean',
    Elected: "N",
    HistoricElected: "N",
    PartyAb: 'LP',
    OrdinaryVotes: 804,
    Swing: -4.49
  },
  {
    PollingPlaceID: 8761,
    DivisionNm: 'Bean',
    Elected: "Y",
    HistoricElected: "N",
    PartyAb: 'ALP',
    OrdinaryVotes: 1405,
    Swing: 4.49
  }
]